docs/reference/intro.asciidoc

```diff
@@ -146,90 +170,197 @@ A mapping defines the <<mapping-types,data type>> for each field, how the field
 and how it should be stored.
 When adding documents to {es}, you have two options for mappings:
 
-* <<mapping-dynamic, Dynamic mapping>>: Let {es} automatically detect the data types and create the mappings for you. This is great for getting started quickly, but can lead to unexpected results for complex data.
-* <<mapping-explicit, Explicit mapping>>: Define the mappings up front by specifying data types for each field. Recommended for production use cases, because you have much more control over how your data is indexed.
+* <<mapping-dynamic, Dynamic mapping>>: Let {es} automatically detect the data types and create the mappings for you. This is great for getting started quickly, but might yield suboptimal results for your specific use case due to automatic field type inference.
+* <<mapping-explicit, Explicit mapping>>: Define the mappings up front by specifying data types for each field. Recommended for production use cases, because you have full control over how your data is indexed to suit your specific use case.
 
 [TIP]
 ====
 You can use a combination of dynamic and explicit mapping on the same index.
 This is useful when you have a mix of known and unknown fields in your data.
 ====
 
+// New html page
+[[es-ingestion-overview]]
+=== Add data to {es}
```

Comment:
the changes that you made to this page are so so valuable! I LOVE being able to understand how to get data into ES at a high level. would have been a game changer for March Shaina.

Reply:
The origin of this content has been available for a couple of years, but somewhat hidden: https://www.elastic.co/guide/en/cloud/current/ec-cloud-ingest-data.html 

I've been using this starter content as the foundation of my ingest experience work, which means we'll have some duplication to sort out. We knew that there would be overlaps, so no big surprise there.  We'll figure out the best way to communicate this info to users, and do the right thing.